dev/ProgressBar/ProgressBar.xaml

```diff
@@ -100,162 +149,128 @@
-                                            <DiscreteDoubleKeyFrame KeyTime="0:0:2.51" Value="{ThemeResource ProgressBarIndicatorPauseOpacity}" />
-                                        </DoubleAnimationUsingKeyFrames>
-                                        <ColorAnimationUsingKeyFrames
-                                            Storyboard.TargetName="IndeterminateProgressBarIndicator2"
-                                            Storyboard.TargetProperty="(Shape.Fill).(SolidColorBrush.Color)">
-                                            <LinearColorKeyFrame Value="{ThemeResource SystemAccentColor}" KeyTime="0:0:2.5" />
-                                            <LinearColorKeyFrame Value="{ThemeResource SystemBaseMediumLowColor}" KeyTime="0:0:2.75" />
-                                        </ColorAnimationUsingKeyFrames>
-                                        <FadeOutThemeAnimation TargetName="DeterminateProgressBarIndicator" />
-                                    </Storyboard>
-                                </VisualState>
-                                <VisualState x:Name="Paused">
-                                    <Storyboard>
-                                        <ObjectAnimationUsingKeyFrames
-                                            Storyboard.TargetName="DeterminateProgressBarIndicator"
-                                            Storyboard.TargetProperty="Fill" >
-                                            <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource SystemBaseMediumLowColor}" />
-                                        </ObjectAnimationUsingKeyFrames>
-                                        <DoubleAnimation
-                                            Storyboard.TargetName="DeterminateProgressBarIndicator"
-                                            Storyboard.TargetProperty="Opacity"
-                                            To="{ThemeResource ProgressBarIndicatorPauseOpacity}"
-                                            Duration="0:0:0.25" />
                                     </Storyboard>
                                 </VisualState>
                             </VisualStateGroup>
                         </VisualStateManager.VisualStateGroups>
 
-                        <Border x:Name="ProgressBarRoot"
-                            Background="{TemplateBinding Background}"
+                        <Border
+                            x:Name="ProgressBarRoot"
                             BorderBrush="{TemplateBinding BorderBrush}"
                             BorderThickness="{TemplateBinding BorderThickness}"
+                            Padding="{TemplateBinding Padding}"
                             contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
-                            contract7NotPresent:CornerRadius="{ThemeResource ControlCornerRadius}">
+                            contract7NotPresent:CornerRadius="{StaticResource ProgressBarCornerRadius}">
```

Comment:
If we're using a custom resource, is the contract7NotPresent check necessary?

Reply:
we need to be able to retemplate and reset it for contract7 +, but not without it. this is common treatment of corner radius throughout.

Reply:
When Contract7 is present we apply the corner radius to the control itself. When contract 7 isn't present (rs3 and lower) we need to apply the corner radius to a component which supports it (panels and borders)

---
In reply to: [575494522](https://github.com/microsoft/microsoft-ui-xaml/pull/4158#discussion_r575494522) [](ancestors = 575494522)